lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -122,7 +117,11 @@ def test_credit_requirements_eligible(self):
             response,
             "{}, you have met the requirements for credit in this course.".format(self.USER_FULL_NAME)
         )
-        self.assertContains(response, "Completed by {date}".format(date=self._now_formatted_date()))
+        self.assertContains(response, "Completed by {date}")
+
+        credit_requirements = credit_api.get_credit_requirement_status(self.course.id, self.user.username)
+        for requirement in credit_requirements:
+            self.assertContains(response, requirement['status_date'].strftime('%Y-%m-%d %H:%M'))
```

Comment:
@jcdyer @nasthagiri Here's a completely unambiguous date, taking all the fun out of pushing flaky tests for future developers to find. (Easy enough, and you were both right, just had to find the right datetime to test)